contracts/registry/src/lib.rs

```
@@ -124,6 +134,31 @@ impl RegistryContract {
+            for i in 0..attestors.len() {
+                let candidate = attestors.get_unchecked(i);
+                for j in (i + 1)..attestors.len() {
+                    if candidate == attestors.get_unchecked(j) {
+                        panic_with_error!(&env, Error::DuplicateAttestor);
+                    }
+                }
+            }
+        }
+
+        // 4. Assign the next available ID.
+        let id: u64 = env
```

Clippy (1.97.1)

[warning] unused variable

(warning)